static/app/components/panels/panelTable.tsx

```diff
@@ -19,7 +19,7 @@ type Props = {
    * The body of the table. Make sure the number of children elements are
    * multiples of the length of headers.
    */
-  children: React.ReactNode | (() => React.ReactNode);
+  children?: React.ReactNode | (() => React.ReactNode);
```

Comment:
When would this be the case where children is `undefined`? I would say that it's better UX if `PanelTable` always has `children` (e.g. in the empty case, it should show an empty message)